### PR TITLE
Add Rheobullet's pin to the map

### PR DESCRIPTION
**Pull request by DimitriCloots**
Just started to work with Github
map #2994

### CODE REVIEW

**Comment by hcd2ha** [comment]
🎉

File: _Pins/_rheobullet.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle:Rheobullet
```
Comment (crichID) [inline _Pins/_rheobullet.json]: @DimitriCloots you will need to add a space after each `:` in your file! Please let me know if you need any assistance.

**Comment by crichID** [comment]
@DimitriCloots you will also want to change the directory to all lowercase letters `_pins`!

**Comment by crichID** [comment]
@DimitriCloots I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/2994/commits/90cb82c740ed8482b7f49c27158ac60dd578a2a5. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**